index.html

```diff
@@ -4329,6 +4409,31 @@ <h2 id="examples-tzigla">Tzigla</h2>
 
     <h2 id="changelog">Change Log</h2>
 
+    <b class="header">1.4.0</b> &mdash; <small><i>Apr. 8, 2017</i></small>
+    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.3.3...1.4.0">Diff</a>
+    &mdash; <a href="https://cdn.rawgit.com/jashkenas/backbone/1.4.0/index.html">Docs</a>
+    <br />
+    <ul style="margin-top: 5px;">
```

Comment:
Anyone know what the deal is with this? Why aren't we using a class?

This is unrelated to this pull but we should probably fix separately